flow_action_components/CollectionProcessors/force-app/main/default/classes/SortCollection.cls

```diff
@@ -21,7 +21,7 @@ global with sharing class SortCollection {
             }
```

Comment:
This is great! Welcome to the project. Very glad to have your contribution

Reply:
The Sort has been a very helpful component to a flow that I'm building. Thank you!